Gems/Atom/Feature/Common/Assets/Shaders/Materials/StandardPBR/StandardPBR_Shadowmap_WithPS.shader

```diff
@@ -1,7 +1,10 @@
 {
-    "Source" : "./StandardPBR_Shadowmap_WithPS.azsl",
+    "Source" : "./StandardPBR_DepthPass_WithPS.azsl",
 
-    "DepthStencilState" : { 
+    "Definitions" : ["SHADOWMAP=1"] ,
```

Comment:
This won't be necessary since StandardPBR_Shadowmap_WithPS.azsl already sets this flag.

Reply:
Actually going to delete StandardPBR_Shadowmap_WithPS.azsl since it's the same file as the depth version just with that define, so by putting the define in the shader I can get rid of the .azsl